readthedocs/builds/models.py

```diff
@@ -662,12 +636,33 @@ class Build(models.Model):
     output = models.TextField(_('Output'), default='', blank=True)
     error = models.TextField(_('Error'), default='', blank=True)
     exit_code = models.IntegerField(_('Exit code'), null=True, blank=True)
+
+    # Metadata from were the build happened.
     commit = models.CharField(
         _('Commit'),
         max_length=255,
         null=True,
         blank=True,
     )
+    version_slug = models.CharField(
```

Comment:
We should have a comment above these fields explaining why they are stored here as well as having a `version` relation.